crates/holochain_sqlite/src/sql/cell/schema/6-up.sql

```diff
@@ -0,0 +1,95 @@
+CREATE TABLE Warrant (
+  hash BLOB PRIMARY KEY ON CONFLICT IGNORE,
+  author BLOB NOT NULL,
+  timestamp INTEGER NOT NULL,
+  warrantee BLOB NOT NULL,
+  type TEXT NOT NULL,
+  blob BLOB NOT NULL
+);
+
+-- Drop foreign key constraint on action_hash in DhtOp.
+-- To achieve that, create a new table without it and drop the old table.
```

Comment:
SQLite actually recommends doing all this in a transaction. But that's not been a practice in the other upgrade scripts, and I think it's exaggerated anyway to keep updating the database schema, while there are other breaking changes and a lack of facilities that makes it impossible or impractical to migrate from one version of a conductor to the following.

Reply:
I agree. I think before we release a stable version we should collapse the migrations and start fresh

Reply:
Would it be easier to do that now for this change?

Reply:
Not for me! ;-)